### PR TITLE
chore: update openapi client

### PR DESCRIPTION
**Pull request by mederka**
Depends on: https://github.com/fal-ai/fal-isolate-cloud/pull/1185

### CODE REVIEW

File: .github/workflows/integration_tests.yaml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
 
       - uses: actions/setup-python@v4
         with:
-          python-version: "3.9"
+          python-version: "3.10"
```
Comment (chamini2) [inline .github/workflows/integration_tests.yaml:22]: Should we do this change in a different PR?

**Review by chamini2 — APPROVED** [review]

File: .github/workflows/integration_tests.yaml (hunk 1 above)
Comment (mederka) [reply .github/workflows/integration_tests.yaml:22]: makes sense
